.github/labeler.yml

```diff
@@ -1,7 +1,10 @@
 # Add 'repo' label to any root file changes
 repo:
-- '*'
-
+- owner:SunHappyboy
```

Comment:
Please remove this autoreference from this file

Reply:
Please correct it

Reply:
I has remove it ;